### PR TITLE
cloud import: create a troubleshooting section

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Create a troubleshooting section

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/import-parquet-files.md [hunk 1]
```diff
@@ -180,3 +176,15 @@ The following table lists the supported Parquet data types that can be imported
 | SMALLINT UNSIGNED | N/A | INT32 |
 | TINYINT | N/A | INT32 |
 | TINYINT UNSIGNED | N/A | INT32 |
+
+## Troubleshooting
+
+### Resolve warnings during data import
+
+After clicking **Start Import**, if you see a warning message, resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```
Comment (Frank945946) [inline tidb-cloud/import-parquet-files.md:184]:
```suggestion
After clicking **Start Import**, if you see a warning message like "can't find the corresponding source files", resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```

File: tidb-cloud/import-csv-files.md [hunk 2]
```diff
@@ -163,3 +158,15 @@ Note that only `?` and `*` are supported.
...
+
+### Resolve warnings during data import
+
+After clicking **Start Import**, if you see a warning message such as "can't find the corresponding source files", resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
+
+After resolving these issues, you need to import the data again.
+
+### Zero rows in the imported tables
+
+After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched the value you entered in the **Source file name** field. In this case, check whether there are any typos in the **Source file name** field and try again.
```
Comment (Frank945946) [inline tidb-cloud/import-csv-files.md:172]:
```suggestion
After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched in the Bucket URI that you entered. In this case,resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. And import those empty tables again.
```

Comment (Frank945946) [reply tidb-cloud/import-csv-files.md:172]: @okJiang  PTAL

Comment (okJiang) [reply tidb-cloud/import-csv-files.md:172]: And import those empty tables again.

-> And import tables again.

Comment (okJiang) [reply tidb-cloud/import-csv-files.md:172]: rest LGTM

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/import-csv-files.md [hunk 3]
```diff
@@ -163,3 +158,15 @@ Note that only `?` and `*` are supported.
 > **Note:**
 >
 > The URI must contain the data source folder.
+
+## Troubleshooting
+
+### Resolve warnings during data import
+
+After clicking **Start Import**, if you see a warning message such as "can't find the corresponding source files", resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```
Comment (qiancai) [inline tidb-cloud/import-csv-files.md:166]:
```suggestion
After clicking **Start Import**, if you see a warning message such as `can't find the corresponding source files`, resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```

File: tidb-cloud/import-csv-files.md [hunk 4]
```diff
@@ -163,3 +158,15 @@ Note that only `?` and `*` are supported.
...
+
+After resolving these issues, you need to import the data again.
+
+### Zero rows in the imported tables
+
+After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched in the Bucket URI that you entered. In this case, resolve this issue by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. After that, import those tables again.
```
Comment (qiancai) [inline tidb-cloud/import-csv-files.md:172]:
```suggestion
After the import progress shows **Completed**, check the imported tables. If the number of rows is zero, it means no data files matched the Bucket URI that you entered. In this case, resolve this issue by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. After that, import those tables again.
```

File: tidb-cloud/import-parquet-files.md [hunk 5]
```diff
@@ -180,3 +176,15 @@ The following table lists the supported Parquet data types that can be imported
 | SMALLINT UNSIGNED | N/A | INT32 |
 | TINYINT | N/A | INT32 |
 | TINYINT UNSIGNED | N/A | INT32 |
+
+## Troubleshooting
+
+### Resolve warnings during data import
+
+After clicking **Start Import**, if you see a warning message such as "can't find the corresponding source files", resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```
Comment (qiancai) [inline tidb-cloud/import-parquet-files.md:184]:
```suggestion
After clicking **Start Import**, if you see a warning message such as `can't find the corresponding source files`, resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```

File: tidb-cloud/import-parquet-files.md [hunk 6]
```diff
@@ -180,3 +176,15 @@ The following table lists the supported Parquet data types that can be imported
...
+
+After resolving these issues, you need to import the data again.
+
+### Zero rows in the imported tables
+
+After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched in the Bucket URI that you entered. In this case, resolve this issue by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. After that, import those tables again.
```
Comment (qiancai) [inline tidb-cloud/import-parquet-files.md:190]:
```suggestion
After the import progress shows **Completed**, check the imported tables. If the number of rows is zero, it means no data files matched the Bucket URI that you entered. In this case, resolve this issue by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. After that, import those tables again.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**